internal/pkg/state/state.go

```diff
@@ -108,8 +108,9 @@ func newState(options *Options) *State {
 	mappers := []interface{}{
 		variables.NewMapper(mapperContext),
 		schedulerMapper.NewMapper(mapperContext),
+		sharedcode.NewVariablesMapper(mapperContext),
 		relations.NewMapper(mapperContext),
-		sharedcode.NewMapper(mapperContext),
```

Comment:
V tomto PR rozdeleny balicek `mapper.sharedcode` na:
- `mapper.sharedcode.codes` ... riesi ukladanie do `codes` pod-adresara
- `mapper.sharedcode.variables, ... riesi `variables_id`
- su to vlastne 2 samostatne use-cases, spolu to bolo neprehladne (a potrebujem pridat 3 use-case)
- nic ine sa nemenilo, iba popresuvane subory medzi balickami